File: 프로그래머스/1/42748. K번째수/K번째수.py
def solution(array, commands):
    answer = []
    new_list=[]
    for start, end, th in commands:
        # print(start, end, th)
        new_list=array[start-1:end]
        # print(new_list)
        new_list.sort()
        # print(new_list[th-1])
        answer.append(new_list[th-1])
    return answer

File: 프로그래머스/1/42748. K번째수/K번째수.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

vector<int> solution(vector<int> array, vector<vector<int>> commands) {
    vector<int> answer;
    for(auto a:commands){
        int i=a[0]-1;
        int j=a[1];
        int k=a[2];
        vector<int> temp;
        for(int m=i; m<j; m++){
            temp.push_back(array[m]);
        }
        sort(temp.begin(), temp.end());
        answer.push_back(temp[k-1]);
    }
    return answer;
}